### PR TITLE
docs: add warning about MessageCommand

### PR DESCRIPTION
**Pull request by AnotherBrynAlt**
Based on my experience with the manual, and seeing other people in the discord had the issue.

https://discord.com/channels/737141877803057244/737142503043498015/977466262642896896

This change clarifies the need to get Sapphire to load message commands, since by default they will not be. Since message commands are the first example in the guide, it can be confusing.

### CODE REVIEW

File: docs/Guide/getting-started/creating-a-basic-command.mdx [hunk 1]
```diff
@@ -17,6 +17,14 @@ Note that this doesn't create subcommands. We'll cover those in
 
 :::
 
+:::info
+
+Note that by default Sapphire does not load message commands. You will need to construct your `SapphireClient` with
+the `loadMessageCommandListeners` property set to true. See the `SapphireClientOptions` docs.
+[Client Options](https://www.sapphirejs.dev/docs/Documentation/api-framework/interfaces/SapphireClientOptions#loadmessagecommandlisteners)
+
```
Comment (vladfrangu) [inline docs/Guide/getting-started/creating-a-basic-command.mdx:25]:
```suggestion

Note that by default Sapphire does not load the listeners required for processing message commands. You will need to construct your `SapphireClient` with
the `loadMessageCommandListeners` option set to `true`. See more information about the [client options](https://www.sapphirejs.dev/docs/Documentation/api-framework/interfaces/SapphireClientOptions#loadmessagecommandlisteners).

```

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: docs/Guide/getting-started/creating-a-basic-command.mdx [hunk 2]
```diff
@@ -17,6 +17,13 @@ Note that this doesn't create subcommands. We'll cover those in
 
 :::
 
+:::info
+
+Note that by default Sapphire does not load the listeners required for processing message commands. You will need to construct your `SapphireClient` with
+the `loadMessageCommandListeners` option set to `true`. See more information about the [client options](https://www.sapphirejs.dev/docs/Documentation/api-framework/interfaces/SapphireClientOptions#loadmessagecommandlisteners).
```
Comment (favna) [inline docs/Guide/getting-started/creating-a-basic-command.mdx:23]:
```suggestion
Note that by default Sapphire does not load the listeners required for processing message commands. You will need to construct your `SapphireClient` with
the `loadMessageCommandListeners` option set to `true`. For more information see the [`SapphireClientOptions#loadMessageCommandListeners`][load-message-ecommands-option] interface.
```

Add to the bottom of this file:

```
[load-message-ecommands-option] ../../Documentation/api-framework/interfaces/SapphireClientOptions#loadmessagecommandlisteners
```

**Review by favna — CHANGES_REQUESTED** [review]

File: docs/Guide/getting-started/creating-a-basic-command.mdx (hunk 2 above)
Comment (favna) [reply docs/Guide/getting-started/creating-a-basic-command.mdx]: @AnotherBrynAlt You didn't add the link to the bottom of the file as mentioned above

**Review by favna — APPROVED** [review]

File: docs/Guide/getting-started/creating-a-basic-command.mdx [hunk 3]
```diff
@@ -131,3 +139,6 @@ export class PingCommand extends Command {
 [command]: ../../Documentation/api-framework/classes/Command
 [commandoptions]: ../../Documentation/api-framework/interfaces/CommandOptions
 [getting-started]: ./getting-started-with-sapphire
+
+[load-message-commands-option]
+../../Documentation/api-framework/interfaces/SapphireClientOptions#loadmessagecommandlisteners
```
Comment (favna) [inline docs/Guide/getting-started/creating-a-basic-command.mdx:144]:
```suggestion
[getting-started]: ./getting-started-with-sapphire
[load-message-commands-option]: ../../Documentation/api-framework/interfaces/SapphireClientOptions#loadmessagecommandlisteners
```

**Review by favna — APPROVED** [review]

**Pull request merged**